<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIDriveButton</class>
 <widget class="QWidget" name="UIDriveButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>480</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="HFlatPushButton" name="pushButton_2">
     <property name="text">
      <string>環境変数</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HFlatPushButton" name="pushButton_3">
     <property name="text">
      <string>Folder Managerについて</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/icon/AppIcon.ico</normaloff>:/res/icon/AppIcon.ico</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HFlatPushButton" name="pushButton">
     <property name="text">
      <string>Qtについて</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HFlatPushButton</class>
   <extends>QPushButton</extends>
   <header>HFlatPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
